doc/man7/ossl-guide-tls-client-non-block.pod

```diff
@@ -252,13 +252,26 @@ retry. An optional mode does exist (B<SSL_MODE_ACCEPT_MOVING_WRITE_BUFFER>)
 which will configure OpenSSL to allow the buffer being written to change from
 one retry to the next. However, in this case, you must still retry exactly the
 same data - even though the buffer that contains that data may change location.
-See L<SSL_CTX_set_mode(3)> for further details.
+See L<SSL_CTX_set_mode(3)> for further details. As before we write the request
```

Comment:
There is no _before_ in this document.  _As in the TLS tutorials_ was the wording used elsewhere, although I don't really see the need to reference the TLS tutorials but if they are referenced, it should be a link IMO.